integration_tests/tests/consistency/consistency_user_enhanced.sql

```diff
@@ -0,0 +1,48 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{% set exclude_columns = ['avg_age_currently_open_seconds', 'median_age_currently_open_seconds', 'projects'] %}
```

Comment:
why are projects exluded?

Reply:
Same thing here with the string_agg putting the strings together in a random order.